src/IO/Writer/Module/AsyncWriter.cpp

```diff
@@ -0,0 +1,49 @@
+// SPDX-FileCopyrightText: 2024 SeisSol Group
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "AsyncWriter.h"
+
+#include "async/ExecInfo.h"
+#include <IO/Writer/Writer.h>
+#include <cstddef>
+#include <mpi.h>
+#include <mutex>
+#include <string>
+#include <utils/logger.h>
+
+namespace {} // namespace
```

Comment:
An empty namespace, is it left for future work?

Reply:
Not really; no idea how that one got there. :)

It's harmless, but I'll remove it.